pom.xml

```diff
@@ -228,22 +228,39 @@
             </goals>
             <configuration>
               <archive>
-                <manifestEntries>
-                  <Bundle-Version>${parsedVersion.osgiVersion}</Bundle-Version>
-                  <Bundle-Description>${project.description}</Bundle-Description>
-                  <X-Compile-Source-JDK>${maven.compiler.source}</X-Compile-Source-JDK>
-                  <X-Compile-Target-JDK>${maven.compiler.target}</X-Compile-Target-JDK>
-                  <Implementation-Version>${project.version}</Implementation-Version>
-                  <Multi-Release>true</Multi-Release>
-                </manifestEntries>
                 <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
               </archive>
               <skipIfEmpty>true</skipIfEmpty>
             </configuration>
           </execution>
         </executions>
       </plugin>
- 
+
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>bundle-manifest</id>
+            <phase>process-classes</phase>
+            <goals>
+              <goal>manifest</goal>
+            </goals>
+            <configuration>
+              <unpackBundle>true</unpackBundle>
```

Comment:
I'm don't think this is necessary, since you just want to generate the Manifest.

Reply:
> If you are interested I can suggest a change that should be equivalent.

No objection from me either way.

This is outside of my normal knowledge base, I was just trying to progress this issue forward. 

FWIW I used the org.apache.felix version because it had simple docs: https://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html and I didn't know that it wasn't as up to date.